tests/test_dim_reduction.py

```diff
@@ -0,0 +1,100 @@
+import unittest 
+import pandas as pd 
+import numpy as np 
+import os 
+import sys 
+current_path = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(current_path + "/../neasqc_wp61/models/quantum/")
+from dim_reduction import *
+
+class TestDimReduction(unittest.TestCase):
+    
+    def setUp(self):
+        """
+        Generate random dataset for our tests
+        """
+        n_rows = 10 
```

Comment:
Not sure what n_rows means here... is it the number of fake sentences we want? If so, pls find a better variable name. Also then it should be a parameter and not hard-coded.